### PR TITLE
[RHELC-1323] Modify test to honor overridable kernel check, fix oversights

### PR DESCRIPTION
**Pull request by danmyway**
* modify tests
* fix oversights in logging

Jira Issues: [RHELC-1323](https://issues.redhat.com/browse/RHELC-1323)

Checklist

- [x] PR has been tested manually in a VM (either author or reviewer)
- [x] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [x] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [x] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

**Review by pr-watson — APPROVED** [review]
Looks good to me

File: tests/integration/tier0/non-destructive/kernel/test_kernel_check_verification.py [hunk 1]
```diff
@@ -203,10 +203,7 @@ def test_non_latest_kernel_error(kernel, shell, convert2rhel):
             )
         ) as c2r:
             c2r.expect("Check if the loaded kernel version is the most recent")
-            if c2r.expect("IS_LOADED_KERNEL_LATEST:INVALID_KERNEL_VERSION") == 0:
-                c2r.sendcontrol("c")
-            else:
-                assert (
-                    AssertionError
-                ), "Utility did not raise: ERROR - (ERROR) IS_LOADED_KERNEL_LATEST.INVALID_KERNEL_VERSION"
+            c2r.expect_exact("(OVERRIDABLE) IS_LOADED_KERNEL_LATEST:INVALID_KERNEL_VERSION")
```
Comment (kokesak) [inline tests/integration/tier0/non-destructive/kernel/test_kernel_check_verification.py:206]: non-blocking, but you can update the `test_non_latest_kernel_error()` description, it still contains the word "ERROR"

**Comment by kokesak** [comment]
LGTM! Merging

**Pull request merged**